src/components/ui/ProgressPanel.svelte

```diff
@@ -54,19 +54,68 @@
       default: 'View report'
     }),
     CONFORMANCE_LEVEL: $translate('WCAG.COMMON.CONFORMANCE_LEVEL'),
-    REPORT_FOR: $translate('UI.REPORT.REPORT_FOR')
+    REPORT_FOR: $translate('UI.REPORT.REPORT_FOR'),
+    REPORTED_ON: $translate('UI.REPORT.REPORTED_ON'),
+    REPORTED_ON_OF: $translate('UI.REPORT.REPORTED_ON_OF')
   };
 
-  $:  console.log('totalevaluated', totalEvaluated);
   $: conformanceTarget = $scopeStore['CONFORMANCE_TARGET'];
-  $: wgacVersion = $scopeStore['WCAG_VERSION'];
-  $: percentageEvaluated = (totalEvaluated / totalToEvaluate) * 100;
+  $: wcagVersion = $scopeStore['WCAG_VERSION'];
+  $: percentageTotalEvaluated = (totalEvaluated / totalToEvaluate) * 100;
 
   $: principles = [...new Set($wcag.map((a) => a.num.split('.')[0]))];
 
-  $: console.log($assertions);
+  $: upToWcagVersion = wcagVersion === "2.0" ?
+    ["2.0"] : ["2.0", "2.1"];
 
-  let totalEvaluated = 3;
+  $: filteredCriteria = {
```

Comment:
This criteria by principle filtering might be good to add to wcagStore as well.

Reply:
Added as separate task in https://github.com/w3c/wcag-em-report-tool/issues/411